src/opm/input/eclipse/Schedule/Schedule.cpp

```diff
@@ -177,7 +178,9 @@ namespace Opm {
         rptonly(rptonly_summary_section(SUMMARYSection{ deck })),
         gaslift_opt_active(deck.hasKeyword<ParserKeywords::LIFTOPT>()),
         oilVap(vappars_solution_section(SOLUTIONSection{deck},
-                                        runspec.tabdims().getNumPVTTables()))
+                                        runspec.tabdims().getNumPVTTables())),
+        default_bhp_target_prod{ParserKeywords::FBHPDEF::TARGET_BHP::defaultValue * unit::barsa},
+        default_bhp_limit_inj{ParserKeywords::FBHPDEF::LIMIT_BHP::defaultValue * unit::barsa}
```

Comment:
Instead of `p * unit::barsa` you could, optionally, write this as
```C++
UnitSystem::newMETRIC().to_si(UnitSystem::measure::pressure, p) // default values use METRIC conventions
```
instead.  That's decidedly more verbose and probably a tad more expensive, but also makes it more explicit that you (we) want a unit conversion here.

Reply:
sure thing. i opted to not use the initializer list though so at least we only instance once.